packages/cli/src/utils/token/tests/loadTokenUtils.test.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as fs from "node:fs";
+import { afterEach, describe, expect, it, vi } from "vitest";
+import * as tokenModule from "../loadTokenUtils.js";
+
+vi.mock("consola", () => ({
+  consola: {
+    debug: vi.fn(),
+    warn: vi.fn(),
+  },
+}));
+
+vi.mock("node:fs", () => ({
+  readFileSync: vi.fn(),
+}));
+
+const originalEnv = process.env;
+vi.stubGlobal("process", { ...process, env: { ...originalEnv } });
```

Comment:
Should this use the dedicated stubEnv and unstubEnv?
https://vitest.dev/api/vi#vi-stubenv-0-26-0